moped-editor/src/views/projects/projectsListView/helpers.js

```diff
@@ -134,13 +134,22 @@ export const useColumns = ({ hiddenColumns }) => {
     // Also, some columns are dependencies of other columns to render, so we need to include them.
     // Ex. Rendering ProjectStatusBadge requires current_phase_key which is not a column shown in the UI
     // Parent project Id needs the parent project name
-    const columnsNeededToRender = [
+    const columnsNeededForListView = [
       "project_id",
       "current_phase_key",
       "parent_project_name",
     ];
+    // If parent project id is selected we also need to include the parent project name in the export
+    const columnsNeededForExport = columnsShownInUI.includes(
+      "parent_project_id"
+    )
+      ? ["project_id", "parent_project_name"]
+      : ["project_id"];
```

Comment:
this feels kinda messy to me but needed a way to show the `parent_project_name` in the export if the `parent_project_id` column is turned on since when that column is turned out whats visible in the UI is the name of the parent project + link to the project id

Reply:
I think this is great and a big improvement to have these special cases, where we include columns to make related columns work correctly, near each other in the code. Maybe we will have to extend this in the future, but your comment captures what is going on here and will make that easier imo. 🙏